README.md

```diff
@@ -5,94 +5,140 @@
+| [java-security-test](/java-security-test) | [Integration test code snippet](/samples/java-security-usage/src/test/java/com/sap/cloud/security/samples/HelloJavaServletIntegrationTest.java) for web.xml based servlets <br/>  [Integration test code snippet](/samples/java-security-usage-ias/src/test/java/com/sap/cloud/security/samples/ias/HelloJavaServletIntegrationTest.java) for annotation based servlets             |              
 
-### Requirements
-- Java 8 or 11
-- maven 3.3.9 or later
+:bulb: Changes with SAP Java Buildpack 1.26.0
+The former SAP Java Buildpack versions have used deprecated (Spring) Security libraries and had to be updated. As of version 1.26.0 SAP Java Buildpack uses the [`java-security`](/java-security) library. Please consider these (migration) guides:
 
-### Sample
-- See [java-tokenclient-usage](samples/java-tokenclient-usage) for an example.
-- See [spring-security-xsuaa-usage](samples/spring-security-xsuaa-usage) for an example.
+- [MANDATORY: clean-up deprecated dependencies](https://github.com/SAP/cloud-security-xsuaa-integration/blob/main/java-security/Migration_SAPJavaBuildpackProjects.md)
+- [OPTIONAL: Leverage new APIs and features](https://github.com/SAP/cloud-security-xsuaa-integration/blob/main/java-security/Migration_SAPJavaBuildpackProjects_V2.md)
 
-## Token Validation for Java applications
-Application developers requiring authentication and authorization information in their application use the libraries defined in [java-security](./java-security) to obtain token information like user name.
 
-### Requirements
-- Java 8 or 11
-- maven 3.3.9 or later
+#### 2.1.2. Spring Boot web applications
+Developers seeking OAuth2 token validation and access to token information in their Spring Boot applications can benefit from the [spring-security](/spring-security) library. 
+This library streamlines the process of handling token validation for tokens issued by Xsuaa or Identity services and obtaining token details, such as principal and audiences from the security context.
+[spring-security](/spring-security) can be used for identity service alone configurations, as well as for hybrid setup with identity service along with xsuaa service. For exclusive xsuaa setups, consider using the [spring-xsuaa](/spring-xsuaa) library.
+There is also testing utility library at your disposal that can be used for unit and integration tests. It provides [JwtGenerator](/java-security-test/src/main/java/com/sap/cloud/security/test/JwtGenerator.java) to generate custom JWT tokens and it pre-configures a [WireMock](http://wiremock.org/docs/getting-started/) web server to stub outgoing calls to the identity service to fetch JWKS.
```

Comment:
Again, let's provide the name of the testing library here when it gets mentioned.